packages/lfrqa/src/aviary/envs/lfrqa/env.py

```diff
@@ -118,27 +118,46 @@
 )
 
 
+class LFRQAQuestion(MultipleChoiceQuestion):
+    gt_doc_ids: list[int]
+
+    @model_validator(mode="before")
+    @classmethod
+    def _validate_fields(cls, data: dict) -> dict:
+        if data.get("gold_doc_ids") and not data.get("gt_doc_ids"):
+            data["gt_doc_ids"] = data["gold_doc_ids"]
+        if isinstance(data["gt_doc_ids"], str):
+            data["gt_doc_ids"] = data["gt_doc_ids"].strip("[]").split(",")
+            data["gt_doc_ids"] = [int(_id) for _id in data["gt_doc_ids"]]
+        del data["gold_doc_ids"]
+
+        data["ideal_answer"] = data["answer"]
+        del data["answer"]
+
+        data["question_id"] = data["qid"]
+        del data["qid"]
+
+        data["options"] = []
+        data["prompt_without_options"] = True
+
+        return data
+
+
 class LFRQAPairwiseEvalEnv(GradablePaperQAEnvironment[dict]):
     """Environment to evaluate paperqa's vs human's answers on Long Form RAG QA questions."""
 
     def __init__(
         self,
+        query: LFRQAQuestion,
         *args,
-        qid: str | UUID,
-        question: str,
-        human_answer: str,
-        gt_doc_ids: list[int],
         pairwise_eval_llm: LLMModel | str = CommonLLMNames.GPT_4O.value,
         **kwargs,
     ):
-        kwargs["query"] = question
+        kwargs["query"] = query
         kwargs["docs"] = Docs()
         super().__init__(*args, **kwargs)
 
-        self.qid = qid
-        self.question = question
-        self.human_answer = human_answer
-        self.gt_doc_ids = gt_doc_ids
+        self._query: LFRQAQuestion = query  # type: ignore[mutable-override]
```

Comment:
This line is quite weird. If I didn't do it mypy would raise an error when doing
```python
self._query.question_id
```
Because in the superclass `_query` can also have type string.

Is this fix ok or hacky?